lib/active_actions.rb

```diff
@@ -2,13 +2,12 @@
 
 module ActiveActions ; end
 
+require 'bundler'
+Bundler.require
 require 'active_support/all'
 require 'active_model'
 require 'active_record'
-require 'bundler'
 require 'memoist'
-require 'shaped'
```

Comment:
unrelated change; it was causing problems to require `shaped` here. it also gets required via `Bundler.require` (since `shaped` is listed in the Gemfile, which is necessary since it's not distributed via RubyGems), so we were basically requiring it twice.

This doesn't apply to `memoist`, for example, because `memoist` is not listed (directly) in the Gemfile; it's just listed in the gemspec.